docs/src/content/docs/da/guides/project-structure.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Projekt Struktur
+description: Lær hvordan man organiserer filerne i dit Starlight projekt.
+---
+
+Denne guide viser dig hvordan et Starlight projekt skal opbygges og hvordan de forskellige filer i projektet virker.
+
+Starlight projekter følger for det meste den samme fil- og mappestruktur som andre Astro projekter. Se [Astros projektstrukturs dokumentation](https://docs.astro.build/en/core-concepts/project-structure/) for flere detaljer.
+
+## Filer og mapper
+
+- `astro.config.mjs` — Astro konfigurationsfilen; inkluderer Starlight integrationen og konfigurationen.
+- `src/content/config.ts` — Indholdskollektions-konfigurationsfilen; tilføjer Starlights frontmatter-skemaer til dit projekt.
+- `src/content/docs/` — Indholdsfiler. Starlight laver alle `.md`, `.mdx` eller `.mdoc` filer i denne mappe til en side på din hjemmeside.
+- `src/content/i18n/` (valgfri) — Oversættelses data til at støtte [internationaliseringen](/guides/i18n/).
```

Comment:
Same here: we need to make sure links point to the `/da/` version of pages. (Even if these pages aren’t translated yet, Starlight will generate a fallback page.)

```suggestion
- `src/content/i18n/` (valgfri) — Oversættelses data til at støtte [internationaliseringen](/da/guides/i18n/).
```